### PR TITLE
James 3223 - bump dependency version to fix vulnerabilities

### PR DESCRIPTION
**Pull request by remk**

### CODE REVIEW

**Comment by remk** [comment]
> not sure it will pass the CI though :)

Yeah, i was a little bold with the guava update.
Will see :)

**Comment by mbaechler** [comment]
I like how optimistic your are

**Comment by Arsnael** [comment]
Log output lost 

test this please

**Comment by remk** [comment]
For reference the thing that is holding us back to use the latest guava is jcloud.
A PR has been open recently on it : 
https://github.com/apache/jclouds/pull/75

**Comment by chibenwa** [comment]
test this please (build rescheduled)

File: mpt/impl/imap-mailbox/cyrus/pom.xml [hunk 1]
```diff
@@ -62,7 +62,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>20.0</version><!--$NO-MVN-MAN-VER$-->
+            <version>25.1-jre</version><!--$NO-MVN-MAN-VER$-->
```
Comment (rouazana) [inline mpt/impl/imap-mailbox/cyrus/pom.xml:65]: this line is useless now that all components are using the same guava version, no?

**Comment by remk** [comment]
:green_apple:

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**